src/NuGet/Nuget.csproj

```diff
@@ -1,113 +0,0 @@
-    <MainNuspec>$(TargetDir)NServiceBus.Persistence.Sql.nuspec</MainNuspec>
-    <PersistenceDll>$(SolutionDir)SqlPersistence\bin\$(ConfigurationName)\NServiceBus.Persistence.Sql.dll</PersistenceDll>
-    <PersistencePdb>$(SolutionDir)SqlPersistence\bin\$(ConfigurationName)\NServiceBus.Persistence.Sql.pdb</PersistencePdb>
-    <PersistenceXml>$(SolutionDir)SqlPersistence\bin\$(ConfigurationName)\NServiceBus.Persistence.Sql.xml</PersistenceXml>
-    <!-- MsBuild variables -->
-    <NugetBuildMsBuild>$(SolutionDir)NuGetBuild\NugetBuildMsBuild\</NugetBuildMsBuild>
-    <ScriptBuilderTaskDll>$(SolutionDir)ScriptBuilderTask\bin\$(ConfigurationName)\NServiceBus.Persistence.Sql.ScriptBuilderTask.dll</ScriptBuilderTaskDll>
-    <MsBuildTargets>$(ProjectDir)NServiceBus.Persistence.Sql.MsBuild.targets</MsBuildTargets>
-  </PropertyGroup>
-  <Target Name="NuGetBuild" DependsOnTargets="Clean">
-    <MakeDir Directories="$(NugetBuild)" />
-    <!-- ScriptBuilder -->
-    <MakeDir Directories="$(NugetBuildScriptBuilder)" />
-    <Copy SourceFiles="$(ScriptBuilderNuspec)" DestinationFolder="$(NugetBuildScriptBuilder)" />
-    <Copy SourceFiles="$(ScriptBuilderXml)" DestinationFolder="$(NugetBuildScriptBuilder)lib\net452" />
-    <Exec Command="$(IlMerge) /out:&quot;$(NugetBuildScriptBuilder)lib\net452\NServiceBus.Persistence.Sql.ScriptBuilder.dll&quot; &quot;$(ScriptBuilderDll)&quot; &quot;$(Cecil)&quot; /target:library /targetplatform:v4 /internalize" />
-    <PepitaPackage.CreatePackageTask NuGetBuildDirectory="$(NugetBuildScriptBuilder)" MetadataAssembly="$(ScriptBuilderDll)" />
-    <!-- Main Package -->
-    <MakeDir Directories="$(NugetBuildMain)" />
-    <Copy SourceFiles="$(MainNuspec)" DestinationFolder="$(NugetBuildMain)" />
-    <Copy SourceFiles="$(TargetDir)NServiceBus.Persistence.Sql.nuspec" DestinationFolder="$(NugetBuildMain)" />
-    <Copy SourceFiles="$(PersistenceDll)" DestinationFolder="$(NugetBuildMain)lib\net452" />
-    <Copy SourceFiles="$(PersistencePdb)" DestinationFolder="$(NugetBuildMain)lib\net452" />
-    <Copy SourceFiles="$(PersistenceXml)" DestinationFolder="$(NugetBuildMain)lib\net452" />
-    <PepitaPackage.CreatePackageTask NuGetBuildDirectory="$(NugetBuildMain)" MetadataAssembly="$(PersistenceDll)" />
-    <!-- MsBuild Package -->
-    <MakeDir Directories="$(NugetBuildMsBuild)" />
-    <Copy SourceFiles="$(TargetDir)NServiceBus.Persistence.Sql.MsBuild.nuspec" DestinationFolder="$(NugetBuildMsBuild)" />
-    <Exec Command="$(IlMerge) /out:&quot;$(NugetBuildMsBuild)NServiceBus.Persistence.Sql.ScriptBuilderTask.dll&quot; &quot;$(ScriptBuilderTaskDll)&quot; &quot;$(ScriptBuilderDll)&quot; &quot;$(Cecil)&quot; /target:library /targetplatform:v4 /internalize" />
-    <Copy SourceFiles="$(MsBuildTargets)" DestinationFolder="$(NugetBuildMsBuild)build" />
-    <PepitaPackage.CreatePackageTask NuGetBuildDirectory="$(NugetBuildMsBuild)" MetadataAssembly="$(ScriptBuilderTaskDll)" />
-    <!-- ForIntegration -->
```

Comment:
@gbiellem u dropped the ForIntegration. the msbuild task and the runtime dll should be copied to the `$(SolutionDir)ForIntegration`. try building https://github.com/Particular/NServiceBus.Persistence.Sql/blob/develop/src/Integration/Integration.sln